### ISSUE TITLE
Improve default visualization_recipe() for estimate_predicted with multiple continuous predictors

### ISSUE THREAD

**Issue by bwiernik** [easystats/modelbased#189]
The current approach doesn't include uncertainty bands. I also find using `alpha` instead of `color` to map predictor levels a bit hard to follow. 

It would be good to adopt an approach more like `sjPlot::plot_model(type = "int")` and include a smaller number of lines for specific levels of the second predictor. This would also automatically provide a nice interaction plot.

**Comment by DominiqueMakowski** [comment]
Closing this given the overhaul of plotting logic
